.gitignore

```diff
@@ -79,3 +79,4 @@ simulator/sd
 !tests/firmware-v0.0.0.bin.badsig
 !tests/firmware-v0.0.0.bin.sha256.txt
 !tests/firmware-v0.0.0.bin.sig
+i18n/translations/fr-FR.json
```

Comment:
Is not wrong to place this file on gitignore?

Reply:
have no idea how this happened